src/main/webapp/app/overview/course-exams/course-exams.component.html

```diff
@@ -1,4 +1,4 @@
-<ng-container *ngIf="realExamsOfCourse.length > 0">
+<ng-container *ngIf="realExamsOfCourse.length">
```

Comment:
I don't think this is a good change. `> 0` is maybe slightly longer but semantically better.
It would be a different issue if we had an optional before, but this is not the case.

+ the one below